packages/anoncreds/src/AnonCredsApi.ts

```diff
@@ -224,6 +285,26 @@ export class AnonCredsApi {
       )
 
       await this.anonCredsCredentialDefinitionRepository.save(this.agentContext, credentialDefinitionRecord)
+
+      // Store Credential Definition private data (if provided by issuer service)
+      if (credentialDefinitionPrivate) {
+        const credentialDefinitionPrivateRecord = new AnonCredsCredentialDefinitionPrivateRecord({
```

Comment:
Just to check we're on the same page. This will be moved to the specific anoncreds implementation right?

Maybe we can create an `AnonCredsRepository` or `AnonCredsStorageService` or something that implements the storage for the most common use case (so createMasterSecret, getDefaultMasterSecret, etc...) so it becomes easier to keep them consistent between implementations.